### ISSUE TITLE
Can oops.Code() support multiple data types?

### ISSUE THREAD

**Issue by OneSeven** [samber/oops#33]
In many existing projects, most of the code is of int type, and type conversion is required every time it is passed in and read out.

**Comment by samber** [comment]
I agree, but some projects also use strings. When I added `oops.Code(), I hesitated between both types, and concluded that a numeric value could be converted into a string, but not the opposite.

I suppose an `any` value could fill the gap.

**Comment by AndyCokeZero** [comment]
Hi there! I'm checking in to see if there have been any updates on this issue. The proposal for AnyCode() or supporting multiple data types would be a very welcome addition for projects using integer error codes. Is this still on the roadmap?
